docs/constants/navigation.js

```diff
@@ -363,7 +363,6 @@ const general = [
       makePage('eas-update/custom-updates-server.mdx'),
       makePage('eas-update/codepush.mdx'),
       makePage('eas-update/updating-your-app.mdx'),
-      makePage('eas-update/known-issues.mdx'),
```

Comment:
We should add [redirects](https://github.com/expo/expo/tree/main/docs#redirects) for removing this doc. I can understand this might not be a significant one but the SEO might get affected due to this.

Since this page isn't being replaced by anything else, I think either redirecting to EAS Update introduction page (https://docs.expo.dev/eas-update/introduction/) will be great.